src/renderer/entities/network/model/network-model.ts

```diff
@@ -97,6 +97,24 @@ const initConnectionsFx = createEffect((chains: Record<ChainId, Chain>) => {
   Object.keys(chains).forEach((chainId) => chainConnected(chainId as ChainId));
 });
 
+const updateConnectionFx = createEffect((connection: Connection): Promise<Connection | undefined> => {
+  return storageService.connections.put(connection);
+});
+
+type DisconnectParams = {
+  chainId: ChainId;
+  providers: Record<ChainId, ProviderWithMetadata>;
+};
+const disconnectProviderFx = createEffect(async ({ chainId, providers }: DisconnectParams): Promise<ChainId> => {
+  await providers[chainId].disconnect();
+
+  providers[chainId].on('connected', () => undefined);
+  providers[chainId].on('disconnected', () => undefined);
+  providers[chainId].on('error', () => undefined);
```

Comment:
meybe use `noop` function from lodash?

Reply:
I'd like to reduce lodash usage if it's possible